be/src/olap/base_tablet.cpp

```diff
@@ -414,6 +414,8 @@ void BaseTablet::generate_tablet_meta_copy_unlocked(TabletMeta& new_tablet_meta)
 Status BaseTablet::calc_delete_bitmap_between_segments(
         RowsetSharedPtr rowset, const std::vector<segment_v2::SegmentSharedPtr>& segments,
         DeleteBitmapPtr delete_bitmap) {
+    SCOPED_SWITCH_THREAD_MEM_TRACKER_LIMITER(
```

Comment:
delete bitmap use allocator to manage memory?

Reply:
fixed, `calc_delete_bitmap` tracking in specific task